docker/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+SCRIPT_DIR="$(realpath $(dirname $BASH_SOURCE[0]))"
+
+cd "$SCRIPT_DIR"/wine-docker
+docker build --platform linux/amd64 -t wine-docker:devel --build-arg WINE_FLAVOUR=devel .
+
+cd "$SCRIPT_DIR"/pywine
+sed 's|FROM tobix/wine:devel|FROM wine-docker:devel|' Dockerfile > Dockerfile.modified
```

Comment:
We need some kind of sanity check to ensure that the dockerfile was successfully modified. For example, if the original file is changed to use `tobix2/wine:devel`, this `sed` command will have no effect and our build won't be based on the correct image.

A simple solution might be to compare hashes of the two files. Another solution might be to use `diff -q Dockerfile Dockerfile.modified` and check that it's return code (`$?`) is non-zero.